docs/spec_draft.md

```diff
@@ -1337,6 +1338,39 @@ defined and one of the following:
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.is_finite
+
+### Semantics
+
+Performs element-wise check whether the value in `x` is finite (i.e. is neither
++inf, -inf, nor NaN) and produces a `y` tensor.
+
+### Inputs
+
+| Name | Type                          |
+|------|-------------------------------|
+| `x`  | tensor of floating-point type |
+
+### Outputs
+
+| Name | Type                   |
+|------|------------------------|
+| `y`  | tensor of boolean type |
+
+### Constraints
+
+  * (C1) `x` and `y` have the same shape.
+
+### Examples
+
+```mlir
```

Comment:
How about we add a comment here explaining what logical values these physical values correspond to? For example:

```
// Logical values: -Inf, +Inf, NaN, ...
// %x: [0xFF800000, 0x7F800000, 0x7FFFFFFF, -10.0, -0.0, 0.0, 10.0]
```